app/config/locale/templates/email-inner-base.tpl

```diff
@@ -1,4 +1,4 @@
-<p>{{hello}},</p>
```

Comment:
Don't all the templates need to be updated since this comma is removed?

Reply:
Good call, I should actually have the comma the other way around. There was some back and forth with design, so. I sohuld've went the other way.